src/Nethermind/Nethermind.Network.Stats/Model/Node.cs

```diff
@@ -213,6 +213,14 @@ public static NodeClientType RecognizeClientType(string clientId)
             {
                 return NodeClientType.Reth;
             }
+            else if (clientId.Contains(nameof(NodeClientType.Nimbus), StringComparison.OrdinalIgnoreCase))
+            {
+                return NodeClientType.Nimbus;
+            }
+            else if (clientId.Contains(nameof(NodeClientType.EthereumJS), StringComparison.OrdinalIgnoreCase))
+            {
+                return NodeClientType.EthereumJS;
+            }
```

Comment:
Can we do it based on enum and not if-else? (Can use FastEnum)

Reply:
Not sure I see how?

e.g. it wants to turn `geth go1.23.3 linux` to `NodeClientType.Geth` which isn't a straight parse